powertools-validation/src/main/java/software/amazon/lambda/powertools/validation/ValidationUtils.java

```diff
@@ -255,27 +253,26 @@ public static JsonSchema getJsonSchema(String schema, boolean validateSchema) {
 
     private static JsonSchema createJsonSchema(String schema) {
         JsonSchema jsonSchema;
+        SchemaValidatorsConfig config = SchemaValidatorsConfig.builder().formatAssertionsEnabled(true)
+                .preloadJsonSchemaRefMaxNestingDepth(10).build();
```

Comment:
why setting this to 10 (vs 40 as default value: https://github.com/networknt/json-schema-validator/blob/master/doc/config.md) ?

Reply:
Given that it doesn't seem to be possible to customize the configuration I think a more conservative value of 10 is better. A schema with a lot of nested applicators could still potentially use a lot of memory to preload.